test/hotspot/jtreg/containers/docker/TestMemoryWithSubgroups.java

```diff
@@ -46,6 +47,16 @@ public class TestMemoryWithSubgroups {
 
     private static final String imageName = Common.imageName("subgroup");
 
+    static String getEngineInfo(String format) throws Exception {
+        return DockerTestUtils.execute(Container.ENGINE_COMMAND, "info", "-f", format)
+            .getStdout();
+    }
+
+    static boolean isRootless() throws Exception {
+        return (getEngineInfo("{{.Host.Security.Rootless}}").contains("true") ||
+                getEngineInfo("{{.SecurityOptions}}").contains("name=rootless"));
+    }
```

Comment:
Please add a comment why need to check both (i.e. `docker => .SecurityOptions; podman => .Host.Security.Rootless`. Now it's no longer clear why we query two different values.

Reply:
done, added.